Firestore/Source/Local/FSTLocalStore.mm

```diff
@@ -397,14 +398,14 @@ - (BOOL)shouldPersistQueryData:(FSTQueryData *)newQueryData
   return changes > 0;
 }
 
-- (void)notifyLocalViewChanges:(NSArray<FSTLocalViewChanges *> *)viewChanges {
+- (void)notifyLocalViewChanges:(std::vector<LocalViewChanges> &&)viewChanges {
   self.persistence.run("NotifyLocalViewChanges", [&]() {
-    for (FSTLocalViewChanges *viewChange in viewChanges) {
-      for (const DocumentKey &key : viewChange.removedKeys) {
+    for (LocalViewChanges viewChange : viewChanges) {
```

Comment:
This creates an unnecessary copy. Please declare the loop variable as a const reference (`for (const LocalViewChanges& viewChange : viewChanges)`). In general, the loop variable of a range-based for loop should be a reference, with the exception of very small trivially copyable types (such as built-in integers, doubles, etc.).

Reply:
Done.